sway/commands/new_float.c

```diff
@@ -0,0 +1,7 @@
+#include "log.h"
+#include "sway/commands.h"
+
+struct cmd_results *cmd_new_float(int argc, char **argv) {
+	sway_log(SWAY_INFO, "Warning: new_float is deprecated. Use default_floating_border instead.");
```

Comment:
It is probably worth adding the following:
```c
if (config->reading) {
    config_add_swaynag_warning("new_float is deprecated. Use "
            "default_floating_border instead");
}
```

This will show a swaynag warning when included in the config, which is likely more visible to the user.